tests/providers/test_color.py

```diff
@@ -9,6 +9,7 @@
 from faker.providers.color.fa_IR import Provider as FaIrColorProvider
 from faker.providers.color.hy_AM import Provider as HyAmColorProvider
 from faker.providers.color.sk_SK import Provider as SkSkColorProvider
+from faker.providers.color.he_IL import Provider as HeILColorProvider
```

Comment:
one last thing: could you move this import up 2 lines, so they're in alphabetical order?

Reply:
Done